invenio_rdm_migrator/streams/records/table_generators/records.py

```diff
@@ -102,102 +54,90 @@ def __init__(self, parent_cache, communities_cache):
-        if "doi" in rec["json"]["pids"]:
+        if "doi" in record["json"]["pids"]:
             yield PersistentIdentifier(
                 id=_pid_pk(),
                 pid_type="doi",
-                pid_value=rec["json"]["pids"]["doi"]["identifier"],
+                pid_value=record["json"]["pids"]["doi"]["identifier"],
                 status="R",
                 object_type="rec",
-                object_uuid=rec["id"],
+                object_uuid=record["id"],
                 created=now,
                 updated=now,
             )
         # OAI
-        if "oai" in rec["json"]["pids"]:
+        if "oai" in record["json"]["pids"]:
             yield PersistentIdentifier(
                 id=_pid_pk(),
                 pid_type="oai",
-                pid_value=rec["json"]["pids"]["oai"]["identifier"],
+                pid_value=record["json"]["pids"]["oai"]["identifier"],
                 status="R",
                 object_type="rec",
-                object_uuid=rec["id"],
+                object_uuid=record["id"],
                 created=now,
                 updated=now,
             )
 
-        # parent
-        if parent["json"]["id"] not in self.parent_cache:
-            self.parent_cache[parent["json"]["id"]] = dict(
-                id=parent["id"],
-                version=dict(latest_index=rec["index"], latest_id=rec["id"]),
-            )
-            parent_pid = parent["json"]["pid"]
-            # record
-            yield RDMParentMetadata(
-                id=parent["id"],
-                json=parent["json"],
-                created=parent["created"],
-                updated=parent["updated"],
-                version_id=parent["version_id"],
-            )
-            # recid
-            yield PersistentIdentifier(
-                id=parent_pid["pk"],
-                pid_type=parent_pid["pid_type"],
-                pid_value=parent["json"]["id"],
-                status=parent_pid["status"],
-                object_type=parent_pid["obj_type"],
-                object_uuid=parent["id"],
-                created=now,
-                updated=now,
-            )
-            # parent community
-            if "default" in parent["json"]["communities"]:
-                yield RDMParentCommunityMetadata(
-                    community_id=parent["json"]["communities"]["default"],
-                    record_id=parent["id"],
-                    request_id=None,
-                )
-        else:
-            # parent in cache - update version
-            cached_parent = self.parent_cache[parent["json"]["id"]]
-            # check if current record is a new version of the cached one
-            if cached_parent["version"]["latest_index"] < rec["index"]:
-                cached_parent["version"] = dict(
-                    latest_index=rec["index"], latest_id=rec["id"]
-                )
         # record files
-        record_files = data["record_files"]
+        record_files = record["record_files"]
```

Comment:
if I understand correctly, that is wrong as record is set to `transformed_entry["record"]` which doesn't have a `record_files` property...